packages/common/src/api/tan-query/useUpdateCollection.ts

```diff
@@ -89,11 +90,14 @@ export const useUpdateCollection = () => {
       // If the mutation fails, roll back collection data
       if (context?.previousCollection) {
         queryClient.setQueryData(
-          ['collection', playlistId],
+          [QUERY_KEYS.collection, playlistId],
```

Comment:
nice find

Reply:
I think it would be pretty easy to write a lint rule for this, fwiw.